aaronpoweruser.ReadwiseUnofficial/plugin.json

```diff
@@ -0,0 +1,88 @@
+{
+  "COMMENT": "Details on these fields: https://help.noteplan.co/article/67-create-command-bar-plugins",
+  "macOS.minVersion": "10.13.0",
+  "noteplan.minAppVersion": "3.4.0",
+  "plugin.id": "aaronpoweruser.ReadwiseUnofficial",
+  "plugin.name": "📚 Readwise Unofficial",
+  "plugin.version": "0.1.0",
+  "plugin.lastUpdateInfo": "First release",
+  "plugin.description": "A sync plugin for readwise",
+  "plugin.author": "aaronpoweruser",
+  "plugin.dependencies": [],
+  "plugin.script": "script.js",
+  "plugin.url": "https://github.com/NotePlan/plugins/blob/main/aaronpoweruser.ReadwiseUnofficial/README.md",
+  "plugin.changelog": "https://github.com/NotePlan/plugins/blob/main/aaronpoweruser.ReadwiseUnofficial/CHANGELOG.md",
+  "plugin.commands": [
+    {
+      "name": "Readwise sync",
+      "description": "Sync readwise highlights",
+      "jsFunction": "readwiseSync"
+    },
+    {
+      "name": "onOpen",
+      "description": "DO NOT EDIT THIS COMMAND/TRIGGER",
+      "jsFunction": "onOpen",
+      "hidden": true
+    },
+    {
+      "name": "onEditorWillSave",
+      "description": "DO NOT EDIT THIS COMMAND/TRIGGER",
+      "jsFunction": "onEditorWillSave",
+      "hidden": true
+    }
+  ],
+  "plugin.settings": [
+    {
+      "COMMENT": "Plugin settings documentation: https://help.noteplan.co/article/123-plugin-configuration",
+      "type": "heading",
+      "title": "Readwise Unofficial Settings"
+    },
+    {
+      "title": "Access token",
+      "key": "accessToken",
+      "required": true,
+      "type": "string",
+      "description": "Enter your readwise access token from readwise.io/access_token",
+      "default": ""
+    },
+    {
+      "title": "Folder name",
+      "key": "baseFolder",
+      "type": "string",
+      "description": "Enter some string and see it change when the plugin is run",
+      "default": "This default setting was set in plugin preferences!"
+    },
+    {
+      "COMMENT": "TODO: This should be a bool but doesn't work",
+      "title": "Group by type",
+      "key": "groupByType",
+      "type": "string",
+      "description": "Group by content type (article, book, etc)",
+      "default": "true",
```

Comment:
No quotes around "true" I think. But don't you want this to be a "string" type with "choices" of `["article", "book", etc.]`?

Reply:
It should be a bool between having all highlights in under the readwise folder or having them separated by content type ie readwise/books, readwise/articles =